docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotHashEncoding.cs

```diff
@@ -0,0 +1,112 @@
+            var hashEncodedData = bagPipeline.Fit(data).Transform(data);
+
+            // Getting the data of the newly created column, so we can preview it.
+            var encodedColumn = hashEncodedData.GetColumn<float[]>("EducationOneHotHashEncoded");
+
+            Console.WriteLine("One Hot Hash Encoding of single column 'Education', based on the bagging strategy.");
+            foreach (var row in encodedColumn)
+            {
+                for (var i = 0; i < row.Length; i++)
+                    Console.Write($"{row[i]} ");
+                Console.WriteLine();
+            }
+
+            // We have 8 slots, because we used numberOfBits = 3.
+
+            // 0 0 0 1 0 0 0 0
+            // 0 0 0 1 0 0 0 0
+            // 0 0 0 0 1 0 0 0
+            // 0 0 0 0 1 0 0 0
+            // 0 0 0 0 0 0 0 1
+
+            // A pipeline for one hot hash encoding the Education column (using keying strategy).
+            var keyPipeline = mlContext.Transforms.Categorical.OneHotHashEncoding("EducationOneHotHashEncoded", "Education", 
+                outputKind: OneHotEncodingEstimator.OutputKind.Key,
+                numberOfBits: 3);
+
+            // Fit and transform the data.
+            var hashKeyEncodedData = keyPipeline.Fit(data).Transform(data);
+
+            // Getting the data of the newly created column, so we can preview it.
+            var keyEncodedColumn = hashKeyEncodedData.GetColumn<uint>("EducationOneHotHashEncoded");
+
+            Console.WriteLine("One Hot Hash Encoding of single column 'Education', with key type output.");
+            foreach (var element in keyEncodedColumn)
+                Console.WriteLine(element);
+
+            // 4
+            // 4
+            // 5
+            // 5
+            // 8
+
+            // Multi column example : A pipeline for one hot has encoding two columns 'Education' and 'ZipCode' 
+            var multiColumnKeyPipeline = mlContext.Transforms.Categorical.OneHotHashEncoding(
+                new InputOutputColumnPair[] { new InputOutputColumnPair("Education"), new InputOutputColumnPair("ZipCode") },
+                numberOfBits: 3);
+
+            // Fit and Transform the data.
+            var transformedData = multiColumnKeyPipeline.Fit(data).Transform(data);
+
+            var convertedData = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, true);
+
+            Console.WriteLine("One Hot Hash Encoding of two columns 'Education' and 'ZipCode'.");
+            foreach (var item in convertedData)
+                Console.WriteLine("{0}\t\t\t{1}", string.Join(" ", item.Education), string.Join(" ", item.ZipCode));
+
+            // 0 0 0 1 0 0 0 0                 0 0 0 0 0 0 0 1
+            // 0 0 0 1 0 0 0 0                 1 0 0 0 0 0 0 0
+            // 0 0 0 0 1 0 0 0                 0 0 0 0 0 0 0 1
+            // 0 0 0 0 1 0 0 0                 1 0 0 0 0 0 0 0
+            // 0 0 0 0 0 0 0 1                 0 0 0 0 0 0 0 1
+        }
```

Comment:
separate example.

Reply:
created separate example for multi column

---
In reply to: [271811872](https://github.com/dotnet/machinelearning/pull/3179#discussion_r271811872) [](ancestors = 271811872)